.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "gulp",
+
+  "rules": {
+    "max-statements": 0
```

Comment:
I'm going to remove this to keep getting the warnings so we can fix it in the future